fed/proxy/barriers.py

```diff
@@ -28,7 +30,11 @@
 from fed._private import constants
 from fed._private.grpc_options import get_grpc_options, set_max_message_length
 from fed.config import get_cluster_config
-from fed.grpc import fed_pb2, fed_pb2_grpc
+if StrictVersion(importlib.metadata.version('protobuf')) >= StrictVersion('4.0.0'):
```

Comment:
You can use https://github.com/ray-project/rayfed/blob/main/fed/_private/compatible_utils.py#L23C5-L23C29

for version comparing, instead of introducing a new package.